src/components/common/container.module.scss

```diff
@@ -18,4 +18,28 @@
   & .container-narrow {
     max-width: 42rem;
   }
+  &.text-heavy {
+    margin: 5em auto;
+    @media (min-width: $viewport-lg) {
+      font-size: 22px;
+      line-height: 1.5;
+    }
+    h2 {
+      font-size: 30px;
+      line-height: 33px;
```

Comment:
I think my 1.1 leading was a little too aggressive; could you bump this up to 1.25?